packages/plugins/src/hardhat/deployments.ts

```diff
@@ -332,8 +367,6 @@ export const proposeChugSplashBundle = async (
   remoteExecution: boolean,
   ipfsUrl: string
 ) => {
-  await checkValidDeployment(hre.ethers.provider, parsedConfig)
-
```

Comment:
I think we need to put the `checkValidDeployment` / `checkValidUpgrade` calls here instead of in `deployChugSplashConfig` because the latter function is only called in the chugsplash-deploy command, whereas `proposeChugSplashConfig` is called in the chugsplash-propose command as well. I suppose we could also put the logic in `chugsplashCommitTask`, but this way it's only called once per deployment in the proposal step.